vite.config.ts

```diff
@@ -32,6 +32,7 @@ export default defineConfig({
   resolve: {
     alias: [
       { find: "crypto", replacement: "crypto-js" }, // quickjs-emscripten
+      { find: "csv-parse", replacement: "csv-parse/browser/esm" },
```

Comment:
`csv-parse` lib supports multi platform, so I set the alias for the lib for some platform.